test/behavior.zig

```diff
@@ -17,195 +17,201 @@ test {
     _ = @import("behavior/bugs/655.zig");
     _ = @import("behavior/bool.zig");
 
-    if (builtin.zig_backend != .stage2_arm and builtin.zig_backend != .stage2_x86_64) {
-        // Tests that pass for stage1, llvm backend, C backend, wasm backend.
-        _ = @import("behavior/align.zig");
-        _ = @import("behavior/array.zig");
-        _ = @import("behavior/basic.zig");
-        _ = @import("behavior/bitcast.zig");
-        _ = @import("behavior/bugs/624.zig");
-        _ = @import("behavior/bugs/704.zig");
-        _ = @import("behavior/bugs/1486.zig");
-        _ = @import("behavior/bugs/2692.zig");
-        _ = @import("behavior/bugs/2889.zig");
-        _ = @import("behavior/bugs/3046.zig");
-        _ = @import("behavior/bugs/3586.zig");
-        _ = @import("behavior/bugs/4560.zig");
-        _ = @import("behavior/bugs/4769_a.zig");
-        _ = @import("behavior/bugs/4769_b.zig");
-        _ = @import("behavior/bugs/4954.zig");
-        _ = @import("behavior/byval_arg_var.zig");
-        _ = @import("behavior/call.zig");
-        _ = @import("behavior/cast.zig");
-        _ = @import("behavior/defer.zig");
-        _ = @import("behavior/enum.zig");
-        _ = @import("behavior/error.zig");
-        _ = @import("behavior/for.zig");
-        _ = @import("behavior/generics.zig");
-        _ = @import("behavior/if.zig");
-        _ = @import("behavior/import.zig");
-        _ = @import("behavior/incomplete_struct_param_tld.zig");
-        _ = @import("behavior/inttoptr.zig");
-        _ = @import("behavior/member_func.zig");
-        _ = @import("behavior/null.zig");
-        _ = @import("behavior/optional.zig");
-        _ = @import("behavior/pointers.zig");
-        _ = @import("behavior/ptrcast.zig");
-        _ = @import("behavior/ref_var_in_if_after_if_2nd_switch_prong.zig");
-        _ = @import("behavior/src.zig");
-        _ = @import("behavior/struct.zig");
-        _ = @import("behavior/this.zig");
-        _ = @import("behavior/truncate.zig");
-        _ = @import("behavior/try.zig");
-        _ = @import("behavior/undefined.zig");
-        _ = @import("behavior/underscore.zig");
-        _ = @import("behavior/usingnamespace.zig");
-        _ = @import("behavior/void.zig");
-        _ = @import("behavior/while.zig");
+    if (builtin.zig_backend != .stage2_arm) {
+        // Tests that pass for stage1, llvm backend, C backend, wasm backend and x86_64 backend.
+        _ = @import("behavior/align_simple.zig");
+        _ = @import("behavior/array_simple.zig");
```

Comment:
To avoid all this churn in test/behavior.zig I propose that we start doing this strategy:

https://github.com/ziglang/zig/commit/0f3938e498624868a951423a0b1ca9749301130a

Reply:
Sounds great!

Reply:
Done in ab204f8 - have a look and lemme know if this is what you had in mind @andrewrk!